integration-tests/opentelemetry-jdbc-instrumentation/README.md

```diff
@@ -0,0 +1,16 @@
+# OpenTelemetry JDBC instrumentation example
+
+## Running the tests
+
+
+To run the tests in a standard JVM with an Oracle database started as a Docker container, you can run the following command:
+
+```
+mvn verify -Dtest-containers
+```
+
+To also test as a native image, add `-Dnative`:
+
+```
+mvn verify -Dtest-containers -Dnative
```

Comment:
Are you missing the details about the property you need to set to run a specific DB?

Reply:
I added information how to set Oracle/PostgreSQL/MariaDB to a specific version. Thank you for the suggestion.